backend/docs/Glossary/Glossary.md

```diff
@@ -0,0 +1,19 @@
+#### Term 1:
```

Comment:
Wasn't sure where to put this file. If there's a better place for it I'll move it there.